CHANGELOG.md

```diff
@@ -22,11 +22,8 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
-
-  Description goes here (optional)
-
-  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
+- Make radio and checkbox labels `inline-block` instead of `block`
```

Comment:
Can we reword this to explain what it fixes?

"Fixes radio and checkbox labels extending full width of page"